sr_port/repl_filter.c

```diff
@@ -816,15 +817,15 @@ STATICFNDEF int repl_filter_recv_line(char *line, int *line_len, int max_line_le
 					{	/* Half-timeout : take C-stack of the filter program. */
 						half_timeout_done = TRUE;
 						TIMEOUT_DONE(timedout);
-						TIMEOUT_INIT(timedout, FILTER_HALF_TIMEOUT_TIME);
+						TIMEOUT_INIT(timedout, REPL_FILTER_HALF_TIMEOUT);
 						GET_C_STACK_FROM_SCRIPT("FILTERTIMEDOUT_HALF_TIME", process_id, repl_filter_pid, 0);
 					}
 					assert(half_timeout_done);
-					/* GET_C_STACK_FROM_SCRIPT calls gtm_system(BYPASSOK) with interrupts deferredd. If the
-					 * stack trace takes more than 32 seconds, the next timeout interrupt will be deferred
-					 * until gtm_system(BYPASSOK) returns. At which point timedout will be TRUE and there will
-					 * be no signal received by GT.M to interrupt the blocking read() at the begining of the
-					 * loop.  So we handle the timeout now and skip the second stack trace.
+					/* GET_C_STACK_FROM_SCRIPT calls gtm_system(BYPASSOK) with interrupts deferred. If the
+					 * stack trace takes more than REPL_FILTER_HALF_TIMEOUT seconds, the next timeout interrupt
+					 * will be deferred until gtm_system(BYPASSOK) returns. At which point timedout will be
```

Comment:
I'm channeling Roger today:
- "will be deferred" -> "is deferred"
- 2 other "will be"s should be similarly eliminated. I forget what tense it is (active voice?) but Roger beat me about the head and shoulders until I was largely allergic to "will be"s.. :-D